src/javascript/app/pages/cashier/cashier.js

```diff
@@ -247,12 +253,36 @@ const Cashier = (() => {
 
                 if (Currency.isCryptocurrency(currency)) {
                     $('.crypto_currency').setVisibility(1);
+                    $('.change-account-btn-crypto').off('click').on('click', (e)=>{
+                        e.stopPropagation();
+                        $('.main-account').trigger('click');
+                        $('html, body').animate({ scrollTop: 0 }, 'slow');
+                    });
 
                     const previous_href = $('#view_payment_methods').attr('href');
                     $('#view_payment_methods').attr('href', previous_href.concat('?anchor=cryptocurrency'));
                 } else {
                     $('.normal_currency').setVisibility(1);
+                    $('.change-account-btn').off('click').on('click', (e)=>{
+                        e.stopPropagation();
+                        $('.main-account').trigger('click');
+                        $('html, body').animate({ scrollTop: 0 }, 'slow');
+                    });
                 }
+                
+                BinarySocket.wait('authorize').then(() => {
+                    const all_currencies = Client.getAllLoginids().map((loginid) => Client.get('currency', loginid));
```

Comment:
we have a function `getCurrencyValues` that could be exported and used here, it already has a has_fiat flag, we can also add a has_crypto flag to it and use it here

Reply:
can't use it because of
```
 if (!Client.get('is_virtual', loginid) && currency) {
                        other_currencies.push(currency);
```
shall I create a lookalike function, but with different demo-acc behaviour? Or it will be better to leave it as it is now

Reply:
this function is in `src/javascript/app/pages/user/get_currency.js`

Reply:
@Daniil-Deriv noted, alright

Reply:
checked with Negar, will leave it as it is